style.css

```diff
@@ -366,6 +374,161 @@ main {
   color: #fff;
 }
 
+.card {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  inset: 0 0 0 0;
+  background: transparent;
+  backdrop-filter: blur(8px);
+  transform: scale(0);
+  transition: all 500ms ease-in-out;
+  pointer-events: all;
+}
+
+.card.active {
+  transform: scale(1);
+}
+
+.card-details {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  z-index: 100;
+  background: #fff;
+  border-radius: 10px;
+  box-shadow: 0 0 0 1000px rgba(193, 199, 208, 0.9);
+  margin: 0 auto;
+  pointer-events: all;
+  max-width: 400px;
+  padding-block: 3%;
+  padding-inline: 4%;
+  text-align: start;
+}
+
```

Comment:
- I think it will be better if we added some margins to the sides of your card details so that it looks like that of Figma.

Here is what I can see compared to Figma 👇🏽 
⚠️ Kindly ignore the blank image from my Figma screenshot 😉 
|  Your Design  | Figma Design |
| :---:   | :---: | 
| <img width="379" alt="Screenshot 2023-02-08 at 05 00 15" src="https://user-images.githubusercontent.com/99246359/217512622-d5f4312a-f629-4a3d-8d64-904da7a53f52.png"> |!<img width="310" alt="Screenshot 2023-02-08 at 05 08 39" src="https://user-images.githubusercontent.com/99246359/217513241-855eefeb-cc80-4f21-b3de-d35886bd1ad8.png">  |